tools/node.d.ts

```diff
@@ -204,4 +210,915 @@ export class Dictionary {
     size(): number;
 }
 
-export function minify(files: string | string[] | { [file: string]: string }, options?: MinifyOptions): MinifyOutput;
+export function minify(files: string | string[] | { [file: string]: string } | AST_Node, options?: MinifyOptions): MinifyOutput;
+
+declare class AST_Node {
+    constructor(props?: object);
+    static BASE?: AST_Node;
+    static PROPS: string[];
+    static SELF_PROPS: string[];
+    static SUBCLASSES: AST_Node[];
+    static documentation: string;
+    static propdoc?: Record<string, string>;
+    static expressions?: AST_Node[];
+    static warn?: (text: string, props: any) => void;
+    static from_mozilla_ast?: (node: AST_Node) => any;
+    TYPE: string;
+    CTOR: typeof AST_Node;
```

Comment:
CTOR is not an `AST_Node`. Should be: `CTOR: function(props?: object);`

Reply:
I think it is not simple function
```
> node.CTOR.PROPS
[ 'value', 'quote', 'start', 'end' ]
```